### PR TITLE
Dp 794 add translations basic information

### PR DESCRIPTION
**Pull request by dbgoaco**
[DP-794](https://noticingsystem.atlassian.net/jira/software/projects/DP/boards/22?selectedIssue=DP-794)

### CODE REVIEW

**Review by jakzal — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs [hunk 1]
```diff
@@ -17,11 +18,11 @@ public class SupplierVatQuestionModel(IOrganisationClient organisationClient,
     const string VatSchemeName = "VAT";
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs:21]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs [hunk 2]
```diff
@@ -14,11 +15,11 @@ namespace CO.CDP.OrganisationApp.Pages.Supplier;
 public class SupplierWebsiteQuestionModel(IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_WebsiteQuestion_ErrorSelectOption))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs:18]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs [hunk 3]
```diff
@@ -13,7 +14,7 @@ public class LegalFormCompanyActQuestionModel(
     ITempDataService tempDataService) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs:17]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs:1]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs [hunk 4]
```diff
@@ -13,7 +14,7 @@ public class LegalFormLawRegisteredModel(
     IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please enter the law under which your organisation is registered")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormLawRegistered_ErrorMessage))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs:17]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs [hunk 5]
```diff
@@ -13,7 +14,7 @@ public class LegalFormOtherOrganisationModel(
         IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select your organisation's registered legal form")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormOtherOrganisation_PleaseSelect))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs:17]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs [hunk 6]
```diff
@@ -14,7 +15,7 @@ public class LegalFormSelectOrganisationModel(
 {
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```
Comment (dpatel017) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs:18]: Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/AddressPostalSameAsRegisteredQuestion.cshtml
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/AddressPostalSameAsRegisteredQuestion.cshtml:1]: Done.

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml:1]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml:1]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml:1]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml:1]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs (hunk 3 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs (hunk 4 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs (hunk 6 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs [hunk 7]
```diff
@@ -13,7 +14,7 @@ public class LegalFormCompanyActQuestionModel(
     ITempDataService tempDataService) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs:17]:
```suggestion
    [Required(ErrorMessageResourceName = nameof(StaticTextResource.Global_SelectAnOption), ErrorMessageResourceType = typeof(StaticTextResource))]
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml [hunk 8]
```diff
@@ -62,7 +62,7 @@
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
                                     <label class="govuk-label govuk-date-input__label" for="registrationDateDay">
-                                        Day
+                                        @StaticTextResource.Supplier_LegalFormFormationDate_Day
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml:65]: We have `Global_Day` etc which can be used for these

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs [hunk 9]
```diff
@@ -18,18 +19,18 @@ public class LegalFormFormationDateModel(
...
-    [RegularExpression(RegExPatterns.Month, ErrorMessage = "Month must be a valid number")]
+    [Required(ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_MonthErrorMessage))]
+    [RegularExpression(RegExPatterns.Month, ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_MonthInvalidErrorMessage))]
     public string? Month { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Date of registration must include a year")]
-    [RegularExpression(RegExPatterns.Year, ErrorMessage = "Year must be a valid number")]
+    [Required(ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_YearErrorMessage))]
+    [RegularExpression(RegExPatterns.Year, ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_YearInvalidErrorMessage))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs:33]: Please switch to `ErrorMessageResourceName` and `ErrorMessageResourceType` as found elsewhere in validation attributes. What you've done _was_ correct, but we had to update the pattern elsewhere as it didn't work in some circumstances with custom validators. Moving to this pattern allows us to keep it the same everywhere. Translating ErrorMessage directly like this is discouraged by the .Net docs as well

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs [hunk 10]
```diff
@@ -13,7 +14,7 @@ public class LegalFormLawRegisteredModel(
     IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please enter the law under which your organisation is registered")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormLawRegistered_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs:17]: `ErrorMessageResourceName` not `ErrorMessage`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs [hunk 11]
```diff
@@ -13,7 +14,7 @@ public class LegalFormOtherOrganisationModel(
         IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select your organisation's registered legal form")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormOtherOrganisation_PleaseSelect), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs:17]: `ErrorMessageResourceName` / `ErrorMessageResourceType`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs [hunk 12]
```diff
@@ -14,7 +15,7 @@ public class LegalFormSelectOrganisationModel(
 {
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormSelectOrganisation.cshtml.cs:18]: `ErrorMessageResourceName` / `ErrorMessageResourceType`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierEmailAddress.cshtml [hunk 13]
```diff
@@ -21,14 +22,14 @@
...
-                            It should be a shared inbox managed by your team, not your own email address. This will be displayed on notices.
+                            @StaticTextResource.Supplier_EmailAddress_Hint
                         </div>
 
                         <div class="govuk-form-group govuk-!-width-two-thirds @(emailAddressHasError ? "govuk-form-group--error" : "")">
                             @if (emailAddressHasError)
                             {
                                 <p id="email-address-error" class="govuk-error-message">
-                                    <span class="govuk-visually-hidden">Error:</span>
+                                    <span class="govuk-visually-hidden">@StaticTextResource.Supplier_EmailAddress_Error</span>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierEmailAddress.cshtml:32]:
```suggestion
                                    <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierIndividualOrOrg.cshtml [hunk 14]
```diff
@@ -55,14 +56,14 @@
...
-                                    Individual - for example, a sole trader
+                                    @StaticTextResource.Supplier_Individual
                                 </label>
                             </div>
                         </div>
                     </fieldset>
                 </div>
                 <govuk-button>
-                    Continue
+                    @StaticTextResource.Supplier_Continue
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierIndividualOrOrg.cshtml:66]:
```suggestion
                    @StaticTextResource.Global_Continue
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml [hunk 15]
```diff
@@ -1,133 +1,115 @@
...
+                            @ViewData["Title"]
+                        </h1>
+                    </legend>
+                    <div id="waste-hint" class="govuk-hint">
+                        @StaticTextResource.Supplier_OperationQuestion_Hint
+                    </div>
+                    @if (operationTypeHasError)
+                    {
+                    <p id="organisation-type-error" class="govuk-error-message">
+                        <span class="govuk-visually-hidden">@StaticTextResource.Supplier_OperationQuestion_Error</span> @Html.ValidationMessageFor(m => m.SelectedOperationTypes)
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml:39]:
```suggestion
                        <span class="govuk-visually-hidden">@StaticTextResource.Global_Error</span> @Html.ValidationMessageFor(m => m.SelectedOperationTypes)
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml.cs [hunk 16]
```diff
@@ -14,7 +15,7 @@ public class SupplierOperationQuestionModel(IOrganisationClient organisationClie
     private readonly IOrganisationClient organisationClient = organisationClient;
 
     [BindProperty]
-    [NotEmpty(ErrorMessage = "Select at least one option to proceed.")]
+    [NotEmpty(ErrorMessage = nameof(StaticTextResource.Supplier_OperationQuestion_SelectOption), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml.cs:18]: `ErrorMessageResourceName` / `ErrorMessageResourceType`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml [hunk 17]
```diff
@@ -3,13 +3,13 @@
...
 @{
-    ViewData["Title"] = "Are you registered for VAT?";
+    ViewData["Title"] = @StaticTextResource.Supplier_VatQuestion_Title;
     var invalidSelection = ((TagBuilder)Html.ValidationMessageFor(m => m.HasVatNumber)).HasInnerHtml;
     var vatRegHasError = invalidSelection || ((TagBuilder)Html.ValidationMessageFor(m => m.VatNumber)).HasInnerHtml;
 }
 
 @section BeforeContent {
-    <a href="/organisation/@Model.Id/supplier-information/basic-information" class="govuk-back-link">Back</a>
+    <a href="/organisation/@Model.Id/supplier-information/basic-information" class="govuk-back-link">@StaticTextResource.Supplier_Back</a>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml:12]:
```suggestion
    <a href="/organisation/@Model.Id/supplier-information/basic-information" class="govuk-back-link">@StaticTextResource.Global_Back</a>
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml [hunk 18]
```diff
@@ -29,18 +29,18 @@
...
                             <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-vat-yes">
                                 <div class="govuk-form-group @(vatRegHasError ? "govuk-form-group--error" : "")">
                                     <label class="govuk-label" for="vatNumber">
-                                        VAT number
+                                        @StaticTextResource.Supplier_VatQuestion_VatNumberLabel
                                     </label>
                                     <div id="vat-registration-number-hint" class="govuk-hint">
-                                        <!--This is 9 numbers, sometimes with 'GB' at the start, for example 123456789 or GB123456789.-->You can find it
-                                        on your VAT registration certificate.
+                                        <!--This is 9 numbers, sometimes with 'GB' at the start, for example 123456789 or GB123456789.-->
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml:42]: I reckon this comment can go

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs [hunk 19]
```diff
@@ -17,11 +18,11 @@ public class SupplierVatQuestionModel(IOrganisationClient organisationClient,
     const string VatSchemeName = "VAT";
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption), ErrorMessageResourceType = typeof(StaticTextResource))]
     public bool? HasVatNumber { get; set; }
 
     [BindProperty]
-    [RequiredIf(nameof(HasVatNumber), true, ErrorMessage = "Please enter the VAT number.")]
+    [RequiredIf(nameof(HasVatNumber), true, ErrorMessage = nameof(StaticTextResource.Supplier_VatNumber_Error), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs:25]: `ErrorMessageResourceName` / `ErrorMessageResourceType`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml [hunk 20]
```diff
@@ -1,60 +1,61 @@
...
-                </div>
+                    </div>
+
+                </fieldset>
+            </div>
 
-                <govuk-button class="govuk-!-margin-top-4">Save and continue</govuk-button>
-            </form>
-        </div>
+            <govuk-button class="govuk-!-margin-top-4">@StaticTextResource.Supplier_WebsiteQuestion_SaveAndContinue</govuk-button>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml:58]: There was another Save and continue somewhere else wasn't there. I think we should add a `Global_SaveAndContinue`

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs [hunk 21]
```diff
@@ -14,11 +15,11 @@ namespace CO.CDP.OrganisationApp.Pages.Supplier;
 public class SupplierWebsiteQuestionModel(IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_WebsiteQuestion_ErrorSelectOption), ErrorMessageResourceType = typeof(StaticTextResource))]
     public bool? HasWebsiteAddress { get; set; }
 
     [BindProperty]
-    [RequiredIf(nameof(HasWebsiteAddress), true, ErrorMessage = "Enter the website address")]
+    [RequiredIf(nameof(HasWebsiteAddress), true, ErrorMessage = nameof(StaticTextResource.Supplier_WebsiteAddress_ErrorEnterWebsiteAddress), ErrorMessageResourceType = typeof(StaticTextResource))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs:22]: `ErrorMessageResourceName` / `ErrorMessageResourceType`

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml (hunk 18 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml:42]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml (hunk 20 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml:58]: You're right, switched to Global_SaveAndContinue

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs (hunk 7 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs:17]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml (hunk 8 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml:65]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs (hunk 9 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs:33]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs (hunk 10 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs:17]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs (hunk 11 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs:17]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs (hunk 19 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs:25]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs (hunk 21 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs:22]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml (hunk 17 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml:12]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierBasicInformation.cshtml [hunk 22]
```diff
@@ -59,15 +65,15 @@
...
+                        @StaticTextResource.Supplier_SupplierType
                     </dt>
                     <dd class="govuk-summary-list__value">
                         @(details.SupplierType == CO.CDP.Organisation.WebApiClient.SupplierType.Organisation ? "Organisation" : "Individual")
                     </dd>
                     <authorize scope="@OrgScopeRequirement.Editor">
                         <dd class="govuk-summary-list__actions">
                             <a class="govuk-link" href="individual-or-org">
-                                Change
+                                @StaticTextResource.Supplier_Change
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierBasicInformation.cshtml:76]:
```suggestion
                                @StaticTextResource.Global_Change
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml [hunk 23]
```diff
@@ -1,60 +1,61 @@
...
-                </div>
+                    </div>
+
+                </fieldset>
+            </div>
 
-                <govuk-button class="govuk-!-margin-top-4">Save and continue</govuk-button>
-            </form>
-        </div>
+            <govuk-button class="govuk-!-margin-top-4">@StaticTextResource.Global_SaveAndContinue</govuk-button>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml:58]: There's several other instances in your new templates of "Save and continue" - they should all be replaced with this global

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierBasicInformation.cshtml (hunk 22 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierBasicInformation.cshtml:76]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml (hunk 23 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml:58]: Good spot, all updated and removed from resource file.

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierIndividualOrOrg.cshtml (hunk 14 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierIndividualOrOrg.cshtml:66]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierEmailAddress.cshtml (hunk 13 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierEmailAddress.cshtml:32]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml (hunk 15 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierOperationQuestion.cshtml:39]: Done

**Review by andymantell — APPROVED** [review]

**Review by MayuriGoaco — APPROVED** [review]

**Pull request merged**